Blockzilla/URLBar.swift

```diff
@@ -110,6 +112,15 @@ class URLBar: UIView {
         lockIcon.setContentCompressionResistancePriority(UILayoutPriority(rawValue: 1000), for: .horizontal)
         lockIcon.setContentHuggingPriority(UILayoutPriority(rawValue: 1000), for: .horizontal)
         textAndLockContainer.addSubview(lockIcon)
+        
+        pageActionsButton.isHidden = true
+        pageActionsButton.alpha = 0
+        pageActionsButton.setImage(#imageLiteral(resourceName: "icon_page_action"), for: .normal)
+        pageActionsButton.setContentHuggingPriority(UILayoutPriority(rawValue: 1000), for: .horizontal)
```

Comment:
Is there no ".high" we can use instead of a raw value? There may not be in this case--just confirming!

Reply:
UILayoutPriority does offer a "defaultHigh" value. I'm personally against using this for this case as we do not use any of the default___ values in our project for UILayoutPriority, so it would seem to conflict with the project style.